api/v1/helper.go

```diff
@@ -73,6 +77,28 @@ func (e NetFilterType) String() string {
 	}
 }
 
+func InitNicIdMap(client *kubernetes.Clientset, namespace string) error {
+	cm, err := client.CoreV1().ConfigMaps(namespace).Get(
+		context.Background(),
+		SUPPORTED_NIC_ID_CONFIGMAP,
+		metav1.GetOptions{},
+	)
+	// if the configmap does not exist, return false
+	if err != nil {
+		return err
+	}
+	for _, v := range cm.Data {
+		NicIdMap = append(NicIdMap, v)
```

Comment:
Shouldn't we make the existing NicIdMap blank? Its currently populated on Ln 40.

Reply:
oh, forget to remove the old initialization code.